docs/code-review-guidlines.md

```diff
@@ -0,0 +1,29 @@
+# SONiC-MGMT code review guidelines
+
+## Checklist for the PR author
+- The code shall be fully tested before the developer asks for an internal review.
+- The test should pass successfully for 10 iterations.
+- The developer shall make sure there is no degradation after introducing the new code.
+- The code is PEP-8 compliant.
+- Run the pre-commit checker before uploading the PR.
+
+
+## Checklist for the code reviewer
+- If you see something nice in the PR, tell the developer, especially when they addressed one of your comments in a great way. Code reviews often just focus on mistakes, but they should offer encouragement and appreciation for good practices, as well.
+- Look at every line of code that you have been assigned to review.
+- Avoid usage of hardcoded values, define constants instead.
+- Comments are clear and useful, and mostly explain why instead of what.
+- The naming of variables and functions should imply the purpose.
+- Add logs to indicate the state transitions, exceptions, errors, meaningful points, and so on.
+- Log levels should be properly set.
```

Comment:
-  Log levels 
-  Logs to be contained and rotated, otherwise will eat up the host memory.